pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,511 @@
+    echoit "  > Adding possible values none, innodb, crc32 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=innodb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=crc32" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-monitor-disable" ]; then
+    echoit "  > Adding possible values counter, module, pattern and all for option '${OPTION}' to the final list..."
+    echo "${OPTION}=counter" >> ${OUTPUT_FILE}
+    echo "${OPTION}=module" >> ${OUTPUT_FILE}
+    echo "${OPTION}=pattern" >> ${OUTPUT_FILE}
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-monitor-enable" ]; then
+    echoit "  > Adding possible values counter, module, pattern and all for option '${OPTION}' to the final list..."
+    echo "${OPTION}=counter" >> ${OUTPUT_FILE}
+    echo "${OPTION}=module" >> ${OUTPUT_FILE}
+    echo "${OPTION}=pattern" >> ${OUTPUT_FILE}
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-monitor-reset" ]; then
+    echoit "  > Adding possible values counter, module, pattern and all for option '${OPTION}' to the final list..."
+    echo "${OPTION}=counter" >> ${OUTPUT_FILE}
+    echo "${OPTION}=module" >> ${OUTPUT_FILE}
+    echo "${OPTION}=pattern" >> ${OUTPUT_FILE}
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-monitor-reset-all" ]; then
+    echoit "  > Adding possible values counter, module, pattern and all for option '${OPTION}' to the final list..."
+    echo "${OPTION}=counter" >> ${OUTPUT_FILE}
+    echo "${OPTION}=module" >> ${OUTPUT_FILE}
+    echo "${OPTION}=pattern" >> ${OUTPUT_FILE}
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-stats-method" ]; then
+    echoit "  > Adding possible values nulls_equal, nulls_unequal, nulls_ignored for option '${OPTION}' to the final list..."
+    echo "${OPTION}=nulls_equal" >> ${OUTPUT_FILE}
+    echo "${OPTION}=nulls_unequal" >> ${OUTPUT_FILE}
+    echo "${OPTION}=nulls_ignored" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-bin" ]; then
+    echoit "  > Adding possible value binlog for option '${OPTION}' to the final list..."
+    echo "${OPTION}=binlog" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-slow-rate-type" ]; then
+    echoit "  > Adding possible values session, query for option '${OPTION}' to the final list..."  
+    echo "${OPTION}=session" >> ${OUTPUT_FILE}
+    echo "${OPTION}=query" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--replica-type-conversions" ]; then
+    echoit "  > Adding possible values ALL_LOSSY, ALL_NON_LOSSY, ALL_SIGNED, ALL_UNSIGNED for option '${OPTION}' to the final list..."
+    echo "${OPTION}=ALL_LOSSY" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_NON_LOSSY" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_SIGNED" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_UNSIGNED" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--performance-schema-accounts-size" ]; then
```

Comment:
Similary you can club 4 elif blocks here for
--performance-schema-accounts-size
--performance-schema-hosts-size
--performance-schema-max-thread-instances
--performance-schema-users-size

Reply:
Fixed with below statement 
elif [ "${OPTION}" == "--performance-schema-accounts-size" -o "${OPTION}" == "--performance-schema-hosts-size" -o "${OPTION}" == "--performance-schema-max-thread-instances" -o "${OPTION}" == "--performance-schema-users-size" ]; then